packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/TerariumAsset.java

```diff
@@ -28,8 +28,15 @@ public abstract class TerariumAsset implements Serializable {
 	private UUID id = UUID.randomUUID();
 
 	@TSOptional
+	@Column(length = 255)
```

Comment:
Why the limit here? Some Document can have insanely long titles.

Reply:
This was always 255 - we're just explicitly defining it here for visibility. If 255 is too small though this can be upped -512?